modules/installation-vsphere-infrastructure.adoc

```diff
@@ -52,34 +52,34 @@ You must install the {product-title} cluster on a VMware vSphere version 7 insta
 [cols=2, options="header"]
 |===
 |Virtual environment product |Required version
-|VM hardware version | 15 or later
+|VM hardware version | 17 or later
```

Comment:
It should be VMware virtual hardware version 15 or later.

Note: It's "VMware virtual hardware version" and not "VM hardware version"

Reply:
Good catch, I've updated this to 15. I've also updated a few cases in this page to say "VMware virtual hardware version" and "virtual hardware version" instead of just HW version or hardware version.